README.md

```diff
@@ -2,7 +2,7 @@
 Initiative to read research papers.
 
 ## Rules of appending papers:
-1. You should thoroughly read research paper and make proper notes using [https://hackmd.io/](HackMD).
+1. You should thoroughly read research paper and make proper notes using [HackMD](https://hackmd.io/).
 1. HackMD notes must mention those member who were part of reading group
 1. If notes and paper reading is complete append details in table
 1. You can then create pull request to merge this 
```

Comment:
put a full stop at the end of all sentences

Reply:
done